tests/tox_env/python/test_python_api.py

```diff
@@ -220,6 +221,38 @@ def test_python_set_hash_seed_incorrect(tox_project: ToxProjectCreator) -> None:
     assert "tox run: error: argument --hashseed: invalid literal for int() with base 10: 'ok'" in result.err
 
 
+def test_python_use_hash_seed_from_env(tox_project: ToxProjectCreator) -> None:
+    ini = "[testenv]\npackage=skip\ncommands=python -c 'import os; print(os.environ[\"PYTHONHASHSEED\"])'"
+    with patch.dict("os.environ", {"PYTHONHASHSEED": "13"}):
+        result = tox_project({"tox.ini": ini}).run("r", "-e", "py")
```

Comment:
To test this change you don't need to run, please use `tox c -e py -k set_env` as a better proxy 👍 and faster tests.

Reply:
I tried to follow the existing practice, and all 6 tests above it do it in this way. Am I missing something?

Reply:
Those validate that the environment variables are forwarded correctly, with these extra tests is enough to validate that set_env is set correctly.